README.rst

```diff
@@ -0,0 +1,184 @@
+|Build Status| |Documentation Status| |codecov|
```

Comment:
What's the motivation to switching to rst from md?  Does pypi require that?

Either way, this README needs to be updated before we upload.